test/powershell/engine/Basic/CommandDiscovery.Tests.ps1

```diff
@@ -205,4 +205,10 @@ Describe "Command Discovery tests" -Tags "CI" {
             }
         }
     }
+
+    Context "error cases" {
+        It 'Get-Command "less `"-PsPage %db?B of %DoesNotExist:`"" should throw Drive not found' {
+            {Get-Command -Name "less `"-PsPage %db?B of %DoesNotExist:`""} | Should -Throw -ErrorId 'DriveNotFound' -because "The drive 'DoesNotExist:' should not exist"
```

Comment:
```suggestion
            {Get-Command -Name "less `"-PsPage %db?B of %DoesNotExist:`""} | Should -Throw -ErrorId 'DriveNotFound' -Because "The drive 'DoesNotExist:' should not exist"
```